docs/canarytokens/factory.md

```diff
@@ -57,12 +68,67 @@ endpoints:
       - name: kind
         required: true
         type: string
-        description: Specifies the type of Canarytoken (currently only supports aws-id)
+        description:  Specifies the type of Canarytoken. Supported classes include
+                      http, dns, cloned-web, doc-msword, web-image, windows-dir,
+                      pdf-acrobat-reader, msword-macro, msexcel-macro, aws-id, qr-code,
+                      svn, sql, fast-redirect, slow-redirect, slack-api
+      - name: web_image
+        required: false
+        type: string
+        description: An image file for use with web-image tokens (request must be multipart/form-data encoded
+                     if parameter is present, required when using web-image)
+      - name: cloned_web
+        required: false
+        type: string
+        description: Domain to check against (required when creating cloned-web tokens)
+      - name: browser_scanner_enabled
+        required: false
+        type: boolean
+        default: true
+        description: Enables a Javascript scanner to retrieve more information
+                     (only valid with 'http' Canarytokens)
       - name: aws_id_username
         required: false
         type: string
-        description: AWS ID Username is optional if you want to create an AWS API key linked to 
+        description: AWS ID Username is optional if you want to create an AWS API key linked to
                      certain Name otherwise it is randomly generated.
+      - name: browser_redirect_url
+        required: false
+        type: string
+        description: Browser redirect URL is the URL you want your Canarytoken server to redirect attackers to after they have
+                     triggered your Canarytoken token (required when creating fast-redirect and slow-redirect tokens)
+      - name: trigger_type
+        required: false
+        type: string
+        description: SQL action that triggers the SQL Canarytoken - these can be `SELECT`, `INSERT`, `UPDATE` or `DELETE`
+                     (required when creating sql tokens)
+      - name: trigger_name
+        required: false
+        type: string
+        description: Name of the trigger we will create (required when creating sql tokens with `trigger_type` as `INSERT`, `UPDATE` or `DELETE`)
+      - name: table_name
+        required: false
+        type: string
+        description: Name of table that will activate the trigger when the `trigger_type` action is executed on this table (required when creating
+                     sql tokens with `trigger_type` as `INSERT`, `UPDATE` or `DELETE`)
+        expected_values: zxc, asd, qwe
```

Comment:
I'm presuming there are no expected values here (especially `zxc` being one of them)?

Reply:
Ah let me remove that